rlberry/agents/agent.py

```diff
@@ -23,10 +23,10 @@
 class Agent(ABC):
     """Basic interface for agents.
 
-    If the 'inherited class' from Agent use the torch lib, that is higly recommanded to inherit :class:`~rlberry.agents.AgentTorch` instead.
+    If the 'inherited class' from Agent use the torch lib, that is highly recommended to inherit :class:`~rlberry.agents.AgentTorch` instead.
```

Comment:
That is highly recommended -> It is highly recommended...

Reply:
done !